config/routes.rb

```diff
@@ -3,7 +3,7 @@
   # content-store (by collections-publisher) - whenever the routes below
   # change, also change the routes claimed by collections-publisher.
 
-  get "/", to: redirect(path: :browse)
+  root to: "development#index"
```

Comment:
Not sure what the correct pattern is here, but is there any value of retaining "/" as redirecting to browse (I'm not sure why this redirect was here in the first place given "/" will always be routed to frontend), and having the development controller at /development? Although I know frontend is different because it does actually render the homepage, I like the pattern it has of this plus only showing it in development: https://github.com/alphagov/frontend/blob/main/config/routes.rb#L18-L20